java/google/registry/flows/domain/DomainCreateFlow.java

```diff
@@ -221,12 +222,12 @@ public final EppResponse run() throws EppException {
       verifyPremiumNameIsNotBlocked(targetId, now, clientId);
       verifyNoOpenApplications(now);
       verifyIsGaOrIsSpecialCase(tldState, isAnchorTenant);
-    }
-    String signedMarkId = hasSignedMarks
+      signedMarkId = hasSignedMarks
```

Comment:
AFAICT the indentation is correct. You maybe looking at where it was prior, it's now moved up into the if (superuser) block which tabbed it over. And the logic change is intentional as it's the primary purpose of this PR.